tools/kotlin-native-gradle-plugin/src/test/kotlin/ExperimentalPluginTests.kt

```diff
@@ -0,0 +1,393 @@
+package org.jetbrains.kotlin.gradle.plugin.test
+
+import org.gradle.testkit.runner.TaskOutcome
+import org.jetbrains.kotlin.konan.target.HostManager
+import org.jetbrains.kotlin.konan.target.KonanTarget
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import java.io.File
+import java.nio.file.Paths
+import java.util.concurrent.TimeUnit
+import kotlin.test.*
+
+class ExperimentalPluginTests {
+
+    val tmpFolder = TemporaryFolder()
+        @Rule get
```

Comment:
Hides us something to use junit5? 🤔

Reply:
We can switch to JUnit5 and rework this part of tests. But I think it's better to do it in a separate PR.